package.json

```diff
@@ -17,5 +17,6 @@
   },
   "devDependencies": {
     "@vercel/ncc": "0.34.0"
-  }
+  },
+  "packageManager": "yarn@1.22.22+sha512.a6b2f7906b721bba3d67d4aff083df04dad64c399707841b7acf00f6b133b7ac24255f2652fa22ae3534329dc6180534e98d17432037ff6fd140556e2bb3137e"
```

Comment:
@gilbertomaga is this intentional? Honestly this should be removed

Reply:
No, this was added upon yarn install